galaxy-backup_playbook.yml

```diff
@@ -0,0 +1,49 @@
+- hosts: galaxy-backup
+  become: true
+  vars_files:
+      - group_vars/all.yml
+      - group_vars/galaxy_etca.yml
+      - group_vars/VAULT
+      - host_vars/galaxy-backup.yml
+      - secret_group_vars/stats_server_vault
+  pre_tasks:
+    - name: Attach volume to instance
+      include_role:
+        name: attached-volumes
+  vars:
+    - galaxy_backup_ansible_version: 4.0.0a1
+    - galaxy_backup_ansible_venv: /home/ubuntu/venv
+  roles:
+    # - galaxyproject.repos # probably not needed for this VM
+    # - common
+    - geerlingguy.pip
+    # - slg.db-backup # Do not include this role until galaxy production is in Sydney
+    # - dj-wasabi.telegraf # comment this out for now - need to check which db it sends to, and that the db exists on stats
+  post_tasks:
+    - name: Ensure desired version of ansible is installed
+      pip:
+        name: "ansible==4.0.0a1"
```

Comment:
This is a pre-release version. Maybe 4.0.0? Also, it's pretty old, current version id 8.0.0

Reply:
4.0.0 would be OK too. The current version is too new.